services/QuillLMS/engines/evidence/spec/controllers/evidence/feedback_controller_spec.rb

```diff
@@ -102,7 +102,7 @@ module Evidence
         end
 
         it 'should return correct spelling feedback when endpoint returns 200' do
-          stub_request(:get, "https://api.cognitive.microsoft.com/bing/v7.0/SpellCheck?mode=proof&text=#{ERB::Util.url_encode(entry_with_error)}").to_return(:status => 200, :body => { :flaggedTokens => ([{ :token => "spelin" }]) }.to_json, :headers => ({}))
+          stub_request(:get, "https://api.bing.microsoft.com/v7.0/spellcheck?mode=proof&text=#{ERB::Util.url_encode(entry_with_error)}").to_return(:status => 200, :body => { :flaggedTokens => ([{ :token => "spelin" }]) }.to_json, :headers => ({}))
```

Comment:
All of the literal string references shoul use the BING_API_URL constant instead.